config/backend-config/namespace_config_test.go

```diff
@@ -38,7 +38,7 @@ func Test_Namespace_SetUp(t *testing.T) {
 
 func Test_Namespace_Get(t *testing.T) {
 	config.Reset()
-	logger.Init()
+	logger.Reset()
```

Comment:
I don't think we are using any config-specific thing in our logs apart from the log level. 

We can set it to debug in tests, and if spam is a concern, start injecting the logger in components.

 Thus should be able to safely remove `logger.Reset()`

Reply:
As long as the logger package depends on the config package, we may keep it until we reach to the point where we no longer need to depend on reseting the config. I'm not in favour of enabling debug logging in tests, e.g. by exporting some env variable in `make test`. IMO, it should be possible for any test to use it's own config options, including defining the `LOG_LEVEL`.